docs/subnets/create-a-mainnet-subnet.md

```diff
@@ -0,0 +1,662 @@
+Subnet on Mainnet. If you haven't done so, first [Deploy a Subnet on
+Testnet](./create-a-fuji-subnet).
+
+This tutorial shows how to do the following on `Mainnet`.
+
+- Create a Subnet.
+- Deploy a virtual machine based on Subnet-EVM.
+- Add a node as a validator to the Subnet.
+- Join a node to the newly created Subnet.
+
+:::note
+
+All IDs in this article are for illustration purposes only. They are guaranteed to be different in
+your own run-through of this tutorial.
+
+:::
+
+## Prerequisites
+
+- 5+ nodes running and [fully bootstrapped](../nodes/README.md) on `Mainnet`
+- [Avalanche-CLI is installed](install-avalanche-cli) on each validator node's box
+- A [Ledger](https://www.ledger.com/) device
+- You've [created a Subnet configuration](create-evm-subnet-config) and fully tested a [Fuji Testnet
+  Subnet deployment](./create-a-fuji-subnet)
+
+:::warning
+
+Although only one validator is strictly required to run a Subnet, running with less than five
+validators is extremely dangerous and guarantees network downtime. Plan to support at least five
+validators in your production network.
+
+:::
+
+### Getting Your Mainnet NodeIDs
+
+You need to collect the NodeIDs for each of your validators. This tutorial uses these NodeIDs in
+several commands.
+
+To get the NodeID of a `Mainnet` node, call the
+[info.getNodeID](../apis/avalanchego/apis/info.md#infogetnodeid) endpoint. For example:
+
+```text
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.getNodeID"
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+The response should look something like:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "nodeID": "NodeID-5mb46qkSBj81k9g9e4VFjGGSbaaSLFRzD"
+  },
+  "id": 1
+}
+```
+
+In the sample response, `NodeID-5mb46qkSBj81k9g9e4VFjGGSbaaSLFRzD` is the NodeID. Note that the
+`NodeID-` prefix is part of the NodeID.
+
+### Setting up Your Ledger
+
+In the interest of security, all Avalanche-CLI `Mainnet` operations require the use of a connected
+Ledger device. You must unlock your Ledger and run the Avalanche App. See [How to Use
+Ledger](https://support.avax.network/en/articles/6150237-how-to-use-a-ledger-nano-s-or-nano-x-with-avalanche)
+for help getting set up.
+
+Avalanche-CLI supports the Ledger `Nano X`, `Nano S`, and `Nano S Plus`.
+
+Ledger devices support TX signing for any address inside a sequence automatically generated by the device.
+
+By default, Avalanche-CLI uses the first address of the derivation, and that address needs funds to
+issue the TXs to create the Subnet and add validators.
+
+To get the first `Mainnet` address of your ledger device, first make sure it is connected,
+unblocked, and running the Avalanche app. Then execute the `key list` command:
+
+```bash
+avalanche key list --ledger 0 --mainnet
+```
+
+The command first asks the user to authorize the ledger to give the CLI extended public key
```

Comment:
There are mixed ledger and Ledger in the text. Should we use Ledger all the time?

Reply:
I just pushed an update that fixes all non-code differences.